IgOmeProfiling_pipeline.py

```diff
@@ -195,6 +204,7 @@ def run_pipeline(fastq_path, barcode2samplename_path, samplename2biologicalcondi
                  args.left_construct, args.right_construct, args.max_mismatches_allowed, args.min_sequencing_quality, args.minimal_length_required, True if args.gz else False,
                  args.max_msas_per_sample, args.max_msas_per_bc,
                  args.max_number_of_cluster_members_per_sample, args.max_number_of_cluster_members_per_bc,
-                 args.allowed_gap_frequency, concurrent_cutoffs, args.meme_split_size, args.mapitope, args.number_of_random_pssms,
+                 args.allowed_gap_frequency, concurrent_cutoffs, args.meme_split_size, args.mapitope,True if args.stop_random_forest else False,
```

Comment:
Some arguments are boolean, no need for `True if arg else False`.
Fix it for every boolean argument in entire code